server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import reviewVotesRouter from './reviewVotesRouter.js'
+import { Vote } from "../../../models/index.js"
+
+const votesRouter = new express.Router()
+
+votesRouter.get('/', async (req, res) => {
+    try {
+        const votes = await Vote.query()
+        let voteValue
+        if (votes.length === 0) {
+            voteValue = 0
+        }
+        return res.status(201).json({ votes: voteValue })
```

Comment:
uuuuhhh what? Does this even work? 

oh, I see. hmmm I wouldnt call that `votevalue` because its either 0 or undefined....which is entirely strange for an endpoint all together? This endpoint doesnt send back votes for a user?

DESTROY THE ENDPOINT WITH FIRE